datafusion/core/src/physical_plan/file_format/parquet/row_groups.rs

```diff
@@ -347,8 +356,22 @@ mod tests {
         let pruning_predicate = PruningPredicate::try_new(expr, schema.clone()).unwrap();
 
         let schema_descr = get_test_schema_descr(vec![
-            ("c1", PhysicalType::INT32, None, None, None, None),
-            ("c2", PhysicalType::INT32, None, None, None, None),
+            PrimitiveTypeField {
+                name: "c1",
+                physical_ty: PhysicalType::INT32,
+                logical_ty: None,
+                precision: None,
+                scale: None,
+                byte_len: None,
+            },
```

Comment:
When I was reading this, I was thinking these tests might be even more readable if they had a builder style, something like

```rust
 PrimtiveTypeField::new("c1", PhysicalType::INT32)
```

And then for tests that need other optional fields set:

```
 PrimtiveTypeField::new("c1", PhysicalType::INT32)
  .with_logical_type(LogicalType::Decimal {
                scale: 0,
                precision: 9,
            });
```